src/api/lib/backend/api/sources/project.rb

```diff
@@ -40,6 +40,11 @@ def self.key_info(project)
           Backend::Connection.get("/source/#{CGI.escape(project)}/_keyinfo?withsslcert=1&donotcreatecert=1").body
         end
 
+        # Returns the patchinfo for the project
```

Comment:
These changes are related with the commit [7f4ad38](https://github.com/openSUSE/open-build-service/pull/3855/commits/7f4ad38c6c290ff5b41a60946f24f551976f0d18#diff-41a074d4db84de47d12a76dcf6b2aa0eR1l)